.travis.yml

```diff
@@ -16,17 +16,22 @@ jobs:
         - SHA_PR=`git log --oneline -1 | sed -rn "s/.*Merge (.*) into.*/\1/p"`
         - SHA_MAIN=`git log --oneline -1 | sed -rn "s/.*Merge .* into (.*)/\1/p"`
         - scripts/checkpatch.pl --strict --codespell -g $SHA_MAIN..$SHA_PR
+    - name: "sparse check"
+      script:
+        - export ARCH=x86_64
+        - make defconfig
+        - scripts/kconfig/merge_config.sh .config kconfig/base-defconfig kconfig/sof-defconfig
```

Comment:
maybe add hdaudio-defconfig?

Reply:
do you mean use this config
`scripts/kconfig/merge_config.sh .config <PATH>/kconfig/base-defconfig <PATH>/kconfig/sof-defconfig <PATH>/kconfig/hdaudio-codecs-defconfig`

Reply:
yep. But just for sparse checks.